.github/workflows/metallb_e2e.yml

```diff
@@ -0,0 +1,80 @@
+name: MetalLB-E2E
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches:
+      - main
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+"
+defaults:
+  run:
+    shell: bash
+    working-directory: metallboperator
+jobs:
+  main:
+    runs-on: ubuntu-20.04
+    env:
+      built_image: "metallb-operator:ci" # Arbitrary name
+    strategy:
+      matrix:
+        go: ['1.16']
+    name: Go ${{ matrix.go }}
+    steps:
+      - name: Checkout Metal LB Operator
+        uses: actions/checkout@v2
+        with:
+          path: metallboperator
+          fetch-depth: 0 # Fetch all history for all tags and branches
+      - uses: actions/setup-go@v2
+        id: go
+        with:
+          go-version: ${{ matrix.go }}
+      - name: Verify modules
+        run: go mod verify
+      - name: Verify format
+        run: |
+          make fmt
+          git diff --exit-code
+      - name: Verify manifests
+        run: |
+          make manifests
+          git diff --exit-code
+      - name: Checkout MetalLB
+        uses: actions/checkout@v2
+        with:
+          repository: metallb/metallb
+          path: metallb
+          ref: 24b8f9c349866e200ddeda1052b2038f21098b13
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update
```

Comment:
should we run the requirements script here? in case we have new requirements
https://github.com/metallb/metallb/blob/main/dev-env/requirements.txt
the same is done in devscripts for example:
https://github.com/openshift-metal3/dev-scripts/blob/0c4d31ccd762a1a39d6b5a296eb3f402dc1d6e0f/metallb/run_e2e.sh#L42

Reply:
done